bodhi/tests/test_masher.py

```diff
@@ -1009,7 +1028,7 @@ def test_push_timestamps(self, publish, *args):
         publish.assert_called_with(
             topic="mashtask.complete",
             force=True,
-            msg=dict(success=True, repo='f17-updates-testing'))
+            msg=dict(success=True, repo='f17-updates-testing', agent='lmacken'))
```

Comment:
I think that one (where the ``MasherThread`` is created with ``'ralph'``) is in a different test.  This one uses ``self.masher`` which is set to be ``'lmacken'``.